test/tests/modules/io/videoCaptureTests.js

```diff
@@ -45,5 +45,24 @@ module.exports = () => {
         expect(cap.get(cv.CAP_PROP_FRAME_HEIGHT)).to.equal(360);
       });
     });
+
+    describe('set', () => {
+      it('should set properties', () => {
+        const cap = new cv.VideoCapture(getTestVideoPath());
+        const wasSet = cap.set(cv.CAP_PROP_POS_MSEC, 1000)
+        expect(cap.get(cv.CAP_PROP_POS_MSEC)|0).to.equal(1001);
+        expect(wasSet).to.equal(true);
+      });
+    });
+    describe('setAsync', () => {
+      it('should set properties', () => {
+        const cap = new cv.VideoCapture(getTestVideoPath());
+        cap.setAsync(cv.CAP_PROP_POS_MSEC, 1000, (err, wasSet) => {
+           expect(cap.get(cv.CAP_PROP_POS_MSEC)|0).to.equal(1001);
+           expect(wasSet).to.equal(true);
```

Comment:
And invoke it here:

```
expect(wasSet).to.equal(true);
done();
...

```

Reply:
Ah, yes! I totally missed that. Sorry, my bad. 🤦‍♂️